python/orca/src/bigdl/orca/learn/pytorch/experimential/mmcv/mmcv_ray_runner.py

```diff
@@ -138,8 +139,7 @@ def train(self, data_loader, **kwargs):
         self.data_loader = data_loader
         self._max_iters = self._max_epochs * len(self.data_loader)
         self.call_hook('before_train_epoch')
-        metric_meters = AverageMeterCollection()
-        kwargs["metric_meters"] = metric_meters
+        self.log_buffer.clear()
```

Comment:
It seems like LoggerHook is doing this: https://github.com/open-mmlab/mmcv/blob/master/mmcv/runner/hooks/logger/base.py#L145
it this necessary to doing this? for maybe there are some case they want to calculate the overall average log?